### ISSUE TITLE
ignoreDiacritics does not remove combining diacritics

### ISSUE THREAD

**Issue by greendemiurge** [ericgio/react-bootstrap-typeahead#279]
### Version
1.4.2

### Steps to reproduce
* Set an array of options with items that have both combining and composed diacritics:
```JS
var options = [
  {id: 1, name: 'Für with composed diacritic'},
  {id: 2, name: 'Für with combining diacritic'},
];
```
* Search using `fur`

### Expected Behavior
Both options should return

### Actual Behavior
Only the option with the composed diacritic with return (option id 1).

N.B.
https://en.wikipedia.org/wiki/Combining_Diacritical_Marks

**Comment by ericgio** [comment]
For posterity, here's the repro case:

```jsx
const options = [
  'F\u00fcr with composed diacritic.',
  'Fu\u0308r with combined diacritic.',
];
```

**Comment by ericgio** [comment]
This should now be fixed in v2.1.0. Thanks for raising this issue, by the way. It was challenging to solve, but I learned a lot in the process!

**Comment by greendemiurge** [comment]
Thank you so much for resolving this!
